src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -318,4 +319,86 @@ object messages {
         |$code2
         |""".stripMargin
   }
+
+  val implicitClassRestrictionsText =
+    """For a full list of restrictions on implicit classes visit
+      |http://docs.scala-lang.org/overviews/core/implicit-classes.html""".stripMargin
```

Comment:
Is there a highlight available for URLs?  The highlighting on this treats http: as plain text and //restOfTheUrl as a comment.

Also, do highlighted sections compose?